doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -31,9 +57,9 @@ The FIWARE NGSI (Next Generation Service Interface) API defines
 (whether to separate the two interfaces is currently under discussion).
 * a set of typical **roles** played by NGSI-compliant components
 
-### Context data modelling and exchange
+## Terminology
```

Comment:
Note the "Exchanging Context Availability Information", which is at the same section level than the (changed) "Context data modelling and exchange". Thus, probably the structure should be:

```
## Terminology
### Context data modelling and exchange
(subsections using ####)
### Exchanging Context Availability Information
(subsections using ####)
```